manager.go

```diff
@@ -95,6 +95,25 @@ func (m *Manager) LoadConfig(cfgPath string) error {
 	return nil
 }
 
+var ErrMissingCloudCreds = errors.New("can't create NetAppender without cloud creds")
+
+func (m *Manager) NetAppender(ctx context.Context) (*logging.NetAppender, error) {
+	if err := m.dial(ctx); err != nil {
+		return nil, err
+	}
+	if m.cloudAddr == "" || m.partID == "" || m.cloudSecret == "" {
+		return nil, ErrMissingCloudCreds
+	}
+	return logging.NewNetAppender(
+		&logging.CloudConfig{
+			AppAddress: m.cloudAddr,
+			ID:         m.partID,
+			Secret:     m.cloudSecret,
+		},
+		m.conn,
+	)
+}
```

Comment:
As noted in the other comment, I think this could be rolled into dial() to remove some repitition here. E.g. the chance for valid cloud configs is already done by LoadConfig(), so no need to duplicate that again here (once it's in an internal function that can't get called out of order.)